ipv6/address.go

```diff
@@ -0,0 +1,147 @@
+package ipv6
+
+import (
+	"fmt"
+	"net"
+)
+
+const (
+	// SIZE is the number of bits that an IPv6 address takes
+	SIZE int = 128
```

Comment:
This shouldn't be exported. This is another thing I think I changed recently in my obsessing over things.